src/com/hotmail/kalebmarc/textfighter/domain/GameEntity.java

```diff
@@ -0,0 +1,27 @@
+package com.hotmail.kalebmarc.textfighter.domain;
+
+
+public class GameEntity{
+    
+  private static double bankInterest;
+  private static int bankBalance;
+
+  public double getBankInterest(){
```

Comment:
I don't mind the idea of having a `GameEntity` class for general/common things; It's definitely a good idea. However, I don't feel like these bank functions are appropriate here. Since they are specific to banking, they should go in the Bank class.

Reply:
GameEntity will be a POJO for all the classes you have. It is common utility hence its getters and setters are.
Getters Setters are the concepts of Data Hiding in which the actual variables are made private and not exposed to be modified by anyone else.
getBankInterest this is the getter of that variable. This has got not logic mentioned here. This method is used only to retrieve bankInterest Variable which is made private.
Bank class is a service layer which will define the logic of bank specific task. It will use the getter setters from GameEntity by the GameEntity instance declared only in bank class and not in any other class.
For eg : public static GameEntity gameEntity -> By this you will set up the gameInterest values only for gameEntity instance which has the scope only in bank class and nowhere else hence the common utility does not expose any functionality. It is bean which you can use to initialise the variables written inside it